crates/rpc/src/handlers/block.rs

```diff
@@ -0,0 +1,64 @@
+use alloy_primitives::B256;
+use ream_storage::{
+    db::ReamDB,
+    tables::{Field, Table},
+};
+use serde::{Deserialize, Serialize};
+use warp::{
+    http::status::StatusCode,
+    reject::Rejection,
+    reply::{Reply, with_status},
+};
+
+use super::BeaconResponse;
+use crate::types::{errors::ApiError, id::ID};
+
+#[derive(Serialize, Deserialize)]
+struct BlockRootResponse {
+    pub root: B256,
+}
```

Comment:
```suggestion
#[derive(Serialize, Deserialize)]
struct RootResponse {
    pub root: B256,
}
```

Could we just call this `RootResponse` as we are likely to use it in more then one place, so maybe put this in a different file as we will use it for the data as well